src/Components/Articles/Article.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon as Icon } from '@fortawesome/react-fontawesome';
+import { faThumbsUp, faComment, faStar, faBookmark } from '@fortawesome/free-solid-svg-icons';
+
+import getRatings from '../../helpers/getRatings';
+
+/**
+ * Maps articles
+ *
+ * @param {*} props
+ * @returns {object} jsx
+ */
+const Article = props => {
+  let { list } = props;
+  list = list.reverse().slice(0, 10);
+
+  const articles = list.map(
+    ({ id, image, title, description, readTime, favoritesCount, Ratings, commentsCount, slug }) => {
```

Comment:
In our backend database, the `images` is an array of all images used in the article. I think you should take the first image in the array and use it.

Reply:
@Karlmusingo I have implemented the changes.